proto/redpanda/api/dataplane/v1alpha1/topic.proto

```diff
@@ -68,30 +73,30 @@ message CreateTopicRequest {
     // NumPartitions must be -1.
     repeated ReplicaAssignment replica_assignments = 4;
 
-    // Configs is an array of key value config pairs for a topic.
-    // These correspond to Kafka Topic-Level Configs.
+    // An array of key-value config pairs for a topic.
+    // These correspond to Kafka topic-level configs.
     repeated Config configs = 5;
   }
 
-  // Topic is the topic to attempt to create.
+  // The topic to create.
   Topic topic = 1 [(buf.validate.field).required = true];
 
-  // ValidateOnly makes this request a dry-run; everything is validated but
+  // If true, makes this request a dry run; everything is validated but
   // no topics are actually created.
   bool validate_only = 2;
 }
 
 message CreateTopicResponse {
-  // Name is the topic's name.
+  // Name of topic.
   string name = 1;
 
-  // PartitionCount is how many partitions were created for this topic.
-  // This field has a default of -1, which may be returned if the broker
+  // The number of partitions created for the topic.
+  // This field has a default value of -1, which may be returned if the broker
   // does not support v5+ of this request which added support for returning
```

Comment:
@weeco what does "v5+ of this request" refer to and will customers be familiar with it?